lib/shoulda/matchers/active_model/numericality_matchers/comparison_matcher.rb

```diff
@@ -5,12 +5,18 @@ module NumericalityMatchers
         # @private
         class ComparisonMatcher < ValidationMatcher
           ERROR_MESSAGES = {
-            :> => :greater_than,
-            :>= => :greater_than_or_equal_to,
-            :< => :less_than,
-            :<= => :less_than_or_equal_to,
-            :== => :equal_to,
-            :!= => :other_than,
+            :> => { label: :greater_than,
```

Comment:
Can you format this as the following:

``` ruby
{
  label: :greater_than,
  assertions: [false, false, true]
},
...
```

Again, abiding by the ~rule~ guideline that you shouldn't need to add extra spaces to align items.